src/translator/xh_scanner.h

```diff
@@ -80,6 +109,9 @@ class scanner {
   // Consumes <!-- ... -->
   token_type scan_comment();
 
+  // Consumes <?name [attrs]?>
+  token_type scan_pi();
```

Comment:
Why is this called 'pi'?

Reply:
It was named `scan_pi` in the imported codebase. I assume because scan_processing_instruction was too long to type.